handbook/engineering/web/frontend-platform/index.md

```diff
@@ -67,97 +67,55 @@ Here are some of the technologies we use to deliver on our goals:
 
 ## Processes
 
+#### Planning and prioritization
+
+We plan and track our day-to-day work on our [Kanban board](https://github.com/orgs/sourcegraph/projects/144). Our current process (last updated 2021-03-17) is as follows:
+
+- Incoming tickets (e.g. from other teams) arrive in the _Inbox_ column.
+- Work is scheduled by adding a card to either the _Planned (product)_ or the _Planned (eng)_ column.
+   - _Planned (product)_ is for work that directly contributes to product goals, and is refreshed and prioritized by the PM/EM.
+   - _Planned (eng)_ is refreshed and prioritized by the engineers. This is the place for refactorings, developer experience improvements, etc.
+- Work should not be moved into either column until the design and engineering teams agree that it is ready for development.
```

Comment:
Thought: What does this mean? Do we have a sync meeting to discuss these, or is this a casual-slack-message kinda thing?